src/infinite-scroll.js

```diff
@@ -131,7 +129,7 @@ angular.module(MODULE_NAME, [])
         return throttled;
       }
 
-      const handler = (THROTTLE_MILLISECONDS != null) ?
```

Comment:
move the should throttle or not logic into the throttle function, then we can just do:

const handler = throttle(unthrottledHandler, THROTTLE_MILLISECONDS)